src/mito2/src/region.rs

```diff
@@ -168,10 +168,17 @@ impl MitoRegion {
         if writable {
             // Only sets the region to writable if it is read only.
             // This prevents others updating the manifest.
-            let _ = self
+            match self
                 .manifest_ctx
                 .state
-                .compare_exchange(RegionState::ReadOnly, RegionState::Writable);
+                .compare_exchange(RegionState::ReadOnly, RegionState::Writable)
+            {
+                Ok(_) => info!("Set region {} to writable", self.region_id),
```

Comment:
Do we call it in a loop (inside the keeper)? Then we might print a lot of this log.